backend/chunker/chunker_internal_test.go

```diff
@@ -686,6 +734,9 @@ func (f *Fs) InternalTest(t *testing.T) {
 	t.Run("MetadataInput", func(t *testing.T) {
 		testMetadataInput(t, f)
 	})
+	t.Run("testBackwardsCompatibility", func(t *testing.T) {
```

Comment:
nitpick: `t.Run("testBackwardsCompatibility", func)` --> `t.Run("BackwardsCompatibility", func)`
(the `test` prefix stays with function name but goes from visible subtest name)